snuba/cli/rust_consumer.py

```diff
@@ -188,6 +188,12 @@
     default=0,
     help="number of milliseconds to wait for the current batch to be flushed by the consumer in case of rebalance",
 )
+@click.option(
+    "--health-check-consumer-groups",
+    default=[],
+    multiple=True,
+    help="Consumer groups for which to use the Snuba Health Check instead of the default Arroyo Health Check",
+)
```

Comment:
why don't you just pass in what healthcheck to use for the consumer that is being launched? e.g. https://github.com/getsentry/snuba/pull/7203

Reply:
each consumer is launched in its own process. so for the metrics consumer you'd need to pass its own consumer group in as an argument.